acoular/base.py

```diff
@@ -158,8 +158,7 @@ def result(self, num=1):
 
 @deprecated_alias({'numchannels': 'num_channels', 'numsamples': 'num_samples'}, read_only=True, removal_version='25.10')
 class TimeOut(SamplesGenerator):
-    """Abstract base class for any signal processing block that receives data from any
-    :attr:`source` domain and returns time domain signals.
+    """Abstract base class receiving from a :attr:`source` and returning time domain signals.
```

Comment:
```suggestion
    """
    Abstract base class receiving from a :attr:`source` and returning time domain signals.
```

Again `GL01 Docstring text (summary) should start in the line immediately after the opening quotes (not in the same line, or leaving a blank line in between)`